src/protections/ProtectionManager.ts

```diff
@@ -392,7 +404,12 @@ export class ProtectionManager {
             const kick = plDefault(powerLevels["kick"], 50);
             const redact = plDefault(powerLevels["redact"], 50);
 
-            const userLevel = plDefault(users[ownUserId], usersDefault);
+            let userLevel;
+            if (creator === ownUserId || (additionalCreators ? additionalCreators.includes(ownUserId) : false)) {
```

Comment:
```suggestion
            if (createEvent["room_version"] === "12" && (creator === ownUserId || additionalCreators?.includes(ownUserId))) {
```

(we also need to check room version, otherwise we support this functionality in all rooms)